examples/create-maps-with-query/create_maps_with_query.bal

```diff
@@ -0,0 +1,24 @@
+import ballerina/io;
+
+type Student record {|
+    readonly int id;
+    string firstName;
+    string lastName;
+    int score;
+|};
+
+public function main() returns error? {
+    table<Student> key(id) students = table [
+            {id: 1, firstName: "John", lastName: "Smith", score: 100},
+            {id: 2, firstName: "Jane", lastName: "Smith", score: 150},
+            {id: 4, firstName: "Fred", lastName: "Bloggs", score: 200},
+            {id: 7, firstName: "Bobby", lastName: "Clark", score: 300},
+            {id: 9, firstName: "Cassie", lastName: "Smith", score: 250}
+        ];
+
+    // The type of the value in the select clause must belong to the tuple type
+    // `[string, T]`, where the type of the constructed value is `map<T>`. 
+    var studentScores = map from var student in students
```

Comment:
instead of `var` shall we use `map<int>`, to be consistent with the comment?

Reply:
If we provide a CET query expression will anyway construct a map without mentioning the query construct type right? Instead, shall we keep this example and also add another sample that creates a map by CET?